rules/always-simplification.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+const R = require('ramda');
+const ast = require('../ast-helper');
+
+const isCalling = ast.isCalling;
+const isBooleanLiteral = ast.isBooleanLiteral;
+
+const report = (instead, prefer) => `\`always(${instead})\` should be simplified to \`${prefer}\``;
+const getAlternative = R.applyTo(R.__, R.compose(R.toUpper, R.head, R.toString));
+
+const create = context => ({
+    CallExpression(node) {
+        const match = isCalling({
+            name: 'always',
+            arguments: R.both(
+                R.propEq('length', 1),
+                R.where({
+                    0: isBooleanLiteral
+                })
+            )
+        });
+
+        if (match(node)) {
+            const instead = node.arguments[0].value;
+
+            context.report({
+                node,
+                message: report(instead, getAlternative(instead))
+            });
+        }
+    }
+});
+
+module.exports = {
+    create,
+    meta: {
+        docs: {
+            description: 'Detects cases where `always` is redundant',
+            recommended: 'error'
```

Comment:
So, every new PR that introduces new rules should have a major update? What version do you suggest using for this?

Reply:
Not necessarily. Only when we change the `recommended` config. So if we would not enable the new rules by default for the `recommended` config it would be a breaking change.
This means new rules should have their `meta.docs.recommended` setting set to `off`.